lib/ssf/ssf_methods.rb

```diff
@@ -58,6 +58,16 @@ def set_name(name)
     end
 
 
+    def set_tag(name, value)
+      self.tags[name] = value
+    end
+
+    def set_tags(tags)
+      if tags.is_a?(Hash)
```

Comment:
Offhand, I don't know how Ruby does this - what's the behavior if `tags` is nil?